app/scripts/controllers/form.js

```diff
@@ -13,22 +13,16 @@ angular.module('ludwig').controller('FormCtrl', function($scope, $http, $state,
     };
 
     $scope.test.expectedResults.forEach(function(expectedResult) {
-        expectedResult.ref = _.find($scope.possibleValues, { id: expectedResult.code });
```

Comment:
FI i tried to merge ref into expectedResult, but ran into error related to Mongo, so I gave up.